controllers/awsAccess.ts

```diff
@@ -0,0 +1,44 @@
+import { PROFILE_SVC_GITHUB_URL } from "../constants/urls";
+import {addUserToGroup, createUser, fetchAwsUserIdByUsername} from "../utils/awsFunctions";
+const dataAccess = require("../services/dataAccessLayer");
+const userDataLevels = require('../constants/userDataLevels');
```

Comment:
Is there any specific reason to use `require` here?
Above lines are using `import` instead of require

Reply:
```
Argument of type '{ discordId: any; level: any; role: string; }' is not assignable to parameter of type '{ discordId?: any; id?: any; username?: any; usernames?: any; query?: any; userdata: any; level?: any; role?: any; userIds?: any; }'
```
It was returning an error like this on using the import, I guess this can be resolved by passing default values to the rest of the fields right?

Reply:
Same here @samarpan1738, @vinit717 can you help us here please

Reply:
@vikhyat187 
I think we aren't able to use import properly with `../services/dataAccessLayer` and `../constants/userDataLevels` because they are using module.exports (which is CJS style) and import is ES6 modules way of importing

Reply:
yes @samarpan1738 we are not exporting the functions like `export function...`